packages/suite/src/components/suite/NavigationBar/index.tsx

```diff
@@ -55,6 +57,30 @@ export const NavigationBar: React.FC = () => {
         setOpened(false);
     };
 
+    let sessionCount = 0;
+    coinjoinAccounts.forEach(({ session }) => {
+        if (session) {
+            sessionCount++;
+        }
+    });
+
+    const coinjoinStatusBars = useMemo(
+        () =>
+            coinjoinAccounts?.map(({ key, session }) => (
+                // if (!session) {
+                //     return;
+                // }
```

Comment:
I guess this is also commented out just for testing purposes. It causes TS error on L#76

Reply:
now I see your comment